app/views/entities/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div id="entity-index">
+    <div id="total-amount">
+      <h2>Total amount: $<%= @entities.sum(:amount) %></h2>
+    </div>
+  <div id="entities">
+    <div id="category-info">
+      <%= image_tag url_for(@group.icon), alt: "category icon" %>
+      <%= @group.name %>
+    </div>
+    <ul>
+      <% @entities.each do |entity| %>
+          <li>
+              <h2>
+                  <%= entity.name %>
+              </h2>
+              <h3>
+                  amount: $<%= entity.amount %>
+              </h3>
+              <h3>
+                <%= image_tag('calendar.png') %> <%= entity.created_at %>
+              </h3>
+          </li>
+      <% end %>
+    </ul>
+  </div>
+  <%= link_to({ controller: 'entities', action: 'new' }) do %>
+    <button>Add new transaction</button>
+  <%end%>
+</div>
```

Comment:
- [ ] Kindly follow the given design for this page as well as per requirements.

Your design
![image](https://github.com/BANCUNGUYE66/budget-app/assets/81932478/46b4d050-19b3-4fcb-aa5d-6798e4c6349a)

Given design
![image](https://github.com/BANCUNGUYE66/budget-app/assets/81932478/5cc719f0-a085-460f-a275-843f61f88908)

- Kindly ensure you follow the layout of elements on your page to match the given design